mungegithub/mungers/submit-queue.go

```diff
@@ -201,6 +202,10 @@ var (
 			Name: "submitqueue_merge_total",
 			Help: "Number of merges done",
 		}),
+		MergeDelay: prometheus.NewGauge(prometheus.GaugeOpts{
+			Name: "submitqueue_merge_delay",
+			Help: "Time since last merge",
```

Comment:
Should be "Time of last merge"-- and maybe s/MergeDelay/LastMergeTime/?

Reply:
Fair enough.